SaveContext.py

```diff
@@ -267,6 +265,8 @@ def give_item(self, item, count=1):
             self.give_health(count)
         elif item == "Bombchu Item":
             self.give_bombchu_item()
+        elif item == "Recovery Heart": # ItemPool.IGNORE_LOCATION
```

Comment:
I'd prefer importing the constant and using it here.

Reply:
I will do this, no need to worry.

Reply:
With that change, it might make sense to preserve the info that this represents a recovery heart, maybe by simply swapping the expression and the comment, to clarify that that item is being handled. The reason I didn't do it this way is that it would lead to an import cycle.